Extension/PesterTask/PesterV10/Pester.ps1

```diff
@@ -110,30 +109,27 @@ if ($ExcludeTag) {
     $Filter.Add('ExcludeTag', $ExcludeTag)
 }
 
-$PesterConfig['Filter'] = $Filter
+$PesterConfig.Filter = $Filter
 
-$CodeCoverage = @{}
 if ($CodeCoverageOutputFile) {
-    $CodeCoverage['Enabled'] = $True
-    $CodeCoverage['OutputFormat'] = "JaCoCo"
+    $PesterConfig.CodeCoverage.Enabled = $True
+    $PesterConfig.CodeCoverage.OutputFormat = "JaCoCo"
 
     if (-not $PSBoundParameters.ContainsKey('CodeCoverageFolder')) {
         $CodeCoverageFolder = $TestFolder
     }
     $Files = Get-ChildItem -Path $CodeCoverageFolder -include *.ps1, *.psm1 -Exclude *.Tests.ps1 -Recurse |
-        Select-object -ExpandProperty Fullname
+    Select-object -ExpandProperty Fullname
```

Comment:
Can you fix this indent back to where it was? I'm guessing VS Code did this automatically with it's formatter.

Reply:
Yes, it was a VSCode formatting issue.  This was fixed as well in my latest commit.